core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -652,7 +652,7 @@ public ServerBuilder gracefulShutdownTimeout(Duration quietPeriod, Duration time
      */
     @Deprecated
     public ServerBuilder blockingTaskExecutor(Executor blockingTaskExecutor) {
-        this.blockingTaskExecutor = requireNonNull(blockingTaskExecutor, "blockingTaskExecutor");
+        blockingTaskExecutor(blockingTaskExecutor, false);
         return this;
```

Comment:
Could be simplified to:

    return blockingTaskExecutor(blockingTaskExecutor, false);